src/v2/services/implementations/AssetsService.ts

```diff
@@ -26,33 +28,51 @@ export class AssetsService implements IAssetsService {
   }
 
   public async transferNativeAsset(param: ParamAssetTransfer): Promise<void> {
-    const transaction = await this.AssetsRepository.getNativeTransferCall(param);
-    const hash = await this.wallet.signAndSend({
-      extrinsic: transaction,
-      senderAddress: param.senderAddress,
-      successMessage: param.successMessage,
-    });
-    param.finalizedCallback(String(hash));
+    const useableBalance = await this.AssetsRepository.getNativeBalance(param.senderAddress);
+    const isBalanceNotEnough =
```

Comment:
Let's change this to `isBalanceEnough`. Having 'Not' in the name makes it less readable.